### ISSUE TITLE
Support for another languages

### ISSUE THREAD

**Issue by kacperpaczos** [soundofgothic/sog-parser#1]
Did you consider support for English and German?

**Comment by VashingMachine** [comment]
Yeah, I was considering that, adding multiple language support would be nice.
Althought there are no plans right now on implementing that in near future.
